android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -385,6 +381,20 @@ private void downloadPageId() {
                     });
     }
 
+    private void storePageName(String feedType, String key) {
+        if(SharedPreferencesUtil.getString(key, null) == null) {
+            RealmList<SocialLink> socialLinks = event.getSocialLinks();
+            RealmResults<SocialLink> page = socialLinks.where().equalTo("name", feedType).findAll();
+            SocialLink socialLink = page.get(0);
```

Comment:
Here check check size of page. return if it is 0. because it will give array index out of bound error if social link is not available.